api/app/utils.py

```diff
@@ -63,6 +72,18 @@ def get_version_info() -> VersionInfo:
         version_json["package_versions"] = manifest_versions
         version_json["image_tag"] = manifest_versions["."]
 
+    if not _is_saas:
+        from users.models import FFAdminUser
+
+        version_json["self_hosted_data"] = {
+            "has_users": FFAdminUser.objects.count() > 0,
+            "has_logins": FFAdminUser.objects.filter(last_login__isnull=False).exists(),
+            "is_bootstrapped": (
+                settings.ALLOW_ADMIN_INITIATION_VIA_CLI is True
+                and FFAdminUser.objects.filter(email=settings.ADMIN_EMAIL).exists()
```

Comment:
Is checking for `ADMIN_EMAIL` reliable? It's not required when deploying with Docker Compose or Helm.

Maybe we could check for an active superadmin/staff user? Or store some data that indicates the bootstrap process was completed? Sorry, don't have much context to suggest something concrete, it's just `ADMIN_EMAIL` that caught my eye.

Reply:
Is there a way to bootstrap without using `ADMIN_EMAIL`?

Reply:
Yes, or by running `python manage.py createsuperuser` from a shell, or by signing up and making yourself a superuser from the database.

Reply:
I don't think we really care about that. We are only trying to check the bootstrap method that we support, which will be deprecated once the new API for onboarding is ready. @matthewelwell  is this correct?